src/api/spec/support/database_cleaner.rb

```diff
@@ -18,7 +18,7 @@
     # For feature test we use deletion instead of transactions because the
     # test suite and the capybara driver do not use the same server thread.
     DatabaseCleaner.strategy = if example.metadata[:type] == :feature || example.metadata[:type] == :migration || example.metadata[:thinking_sphinx] == true
-                                 # Omit truncating what we have set up in db/seeds.rb except users and roles_user
```

Comment:
This comment wasn't true or it changed? I don't see how this changed.

Reply:
Yeah there are way more things we don't truncate. And we *do* truncate `users`.